packages/clients/dish/src/plugins/Gfi/Content.vue

```diff
@@ -186,12 +173,24 @@ export default Vue.extend({
       return this.sachgesamtheit.properties.Foto !== 'Kein Foto gefunden'
     },
   },
+  beforeCreate() {
+    if (this.hasWindowSize && this.hasSmallWidth) {
+      this.setActionButton({
+        component: ActionButton,
+        props: { exportProperty: this.exportProperty },
+      })
+    }
+  },
```

Comment:
While the `v-if` above updates, this does not. I propose using two watchers (on hasWindowSize/hasSmallWidth) that both fire the same method which sets as action button to the GFI plugin either the object in lines 178-181 or `null`, depending on the two variables' new combinatory state (that may, while we're at it, also be a computed value that's a boolean – that way we prevent mistakes on later changes in the logic by funneling all truth through a single value position).

Reply:
Fixed 08acc17d